### PR TITLE
Adamantine golem leadership

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Adamantine golems have special vocal cords that allow them to send
one-way messages to all golems, due to fragments of resonating
adamantine in their heads. Both of these are organs, and can be removed
and put in other species.
add: You can use adamantine vocal cords by prefixing your message with
":x".
del: Xenobiology is no longer capable of making golem runes with plasma.
Instead, inject plasma into the adamantine slime core to get bars of adamantine
which you can then craft into an incomplete golem shell. Add 10 sheets of
suitable material to finish the shell.
experiment: The metal adamantine is also quite valuable when sold via cargo.
/:cl:

Adamantine Golems are the starting Free Golem spawns, and they are
previously just as ordinary as Iron Golems, which are strictly worse
than a variety of other golem materials. This gives them limited one way
telepathy, to act as leaders and communicate to the other golems.

They're vocal cords because it's a suitable fit for the vocal cord code,
and recieving it is organ based because more organs are always fun.

Making "leadership" slave golems is somewhat of a contradiction, and
they're also somewhat easy to spam. This requires xenobio to at least
have some spare materials lying around (remember, you can use metal and
glass or plasteel).

- [x] Requires actual sprites, rather than recoloured existing organs

image

image

### CODE REVIEW

**Comment by XDTM** [comment]
Point out in the info text that you can use .x

**Comment by octareenroon91** [comment]
Neat. Feel like giving plant folk a channel in some future PR?

**Comment by Durkel** [comment]
Why can't we just have ghost roles that are just ways for people to come back into a round and not bloated to fuck with addons and features?

**Comment by Aloraydrel** [comment]
Stupit durkal

**Comment by coiax** [comment]
Someone doesn't understand what Free Golems are for.

**Comment by Kmc2000** [comment]
Do you want some sprites doing?

If you give me a list I can do them (you weren't very specific) unless one of your own spriters wants to take this.

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -477,17 +477,6 @@
 	..()
 
 //Adamantine
-/datum/chemical_reaction/slime/slimegolem
-	name = "Slime Golem"
-	id = "m_golem"
-	required_reagents = list("plasma" = 1)
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: if you're removing this, make the slime golem shell take plasma

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 2]
```diff
@@ -477,17 +477,6 @@
...
-	required_container = /obj/item/slime_extract/adamantine
-	required_other = 1
-
-/datum/chemical_reaction/slime/slimegolem/on_reaction(datum/reagents/holder)
-	new /obj/effect/golemrune(get_turf(holder.my_atom))
-	..()
-
 /datum/chemical_reaction/slime/slimegolem2
 	name = "Slime Golem 2"
```
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: also correct the names/types that refer to it as slimegolem2 now that it's the only one.

**Comment by GunHog** [comment]
Wait, I do not understand something here. If I cannot use golem runes, does this mean that I can only build **free** golems? Does this PR remove slave golems?

**Comment by XDTM** [comment]
golem shells made in xenobio are slaved

**Comment by GunHog** [comment]
Alright, cool! I was just a bit confused. Thanks!

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm (hunk 2 above)
Comment (coiax) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: but stat tracking?

Comment (ChangelingRain) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm]: stat tracking can adjust!

**Comment by fludd12** [comment]
Correct me if I'm wrong, but does this make xenobio spawning an adamantine golem impossible?

**Comment by coiax** [comment]
Yes. That's the intent. Unless the station finds any adamantine sheets.

**Comment by GunHog** [comment]
> Yes. That's the intent. Unless the station finds any adamantine sheets.

Why? The commander thing could be useful if you build a squad or something. Also, it is a little weird and inconsistent to have "adamantine slimes" and "adamantine" cores be unable to make "adamantine" golems. Since the material, I believe, would be useless for anything else, why not include an addy slime core reaction which produces it for use with the shells?

Again, being able to make any type of golem except adamantine from a slime core named "adamantine" does not make sense. Doubly so when they are still not the most desired of golem types.

**Comment by coiax** [comment]
Hmm, consistency is a good point. Making a single piece of adamantine spawn with the iron reaction would be suitable, I think. It fetches a high price in cargo points.

**Comment by GunHog** [comment]
Remove it as an export or use a different method, I did not know it had any worth to cargo. I do feel that we should be able to make addy golems with the same ease as an iron golem. 

The other golems except iron have something special in combat. Addy golems, having no personal combat abilities and just the one-way talking feature means that they would not be sought if they required more than any of the superior/niche golem choices.

**Comment by KorPhaeron** [comment]
This upsets me because Free Golems were supposed to be a "what happens after scientists let golems wander off" and now they cant even make the golems

Meanwhile the special golems that were supposed to be a free golem thing are now science

It is all backwards

**Comment by coiax** [comment]
Okay, adamantine, the metal, is now capable of being produced by injecting admantine slime cores with iron. One sheet of adamantine makes one artificial golem shell, which makes slave golems. It also sells for 7500 credits per sheet as before. Adamantine golems are now possible again, but expensive.

**Comment by coiax** [comment]
@WJohn HURRY UP, THE CITRUS DEATH KNELL IS SOUNDING

**Comment by jughu** [comment]
Adamitine is copyrighted by urist please do not continue and close pr

**Comment by coiax** [comment]
Sprites are done, thank you @WJohn

**Pull request merged**